### PR TITLE
New recipe : Kaptive

### PR DESCRIPTION
**Pull request by Takadonet**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by Takadonet** [comment]
yahoo! Pass on the first try!

**Comment by bgruening** [comment]
@Takadonet do you think my changes are ok. I have skipped py3k builds as this is the preferred conda way.

**Comment by Takadonet** [comment]
@bgruening Your changes are good. Testing locally first then will push out.

**Comment by Takadonet** [comment]
Just realised changes are already here... Waiting for test to pass then.

File: recipes/kaptive/build.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+mkdir -p $PREFIX/bin
+
+cp kaptive.py $PREFIX/bin
+cp extras/kaptive_slurm.py $PREFIX/bin
+
+chmod a+x $PREFIX/bin/kaptive.py
+chmod a+x $PREFIX/bin/kaptive_slurm.py
+
+export PATH=$PREFIX/bin:$PATH
```
Comment (bgruening) [inline recipes/kaptive/build.sh]: Ah sorry, just saw this. This export is not needed here.

**Comment by bgruening** [comment]
Awesome! Thanks a lot @Takadonet!

**Pull request merged**